build/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
@Nikhil-Ladha this is already part of downstream only change in main branch, we might need to delete the sync main branch and close this PR. PTAL

Reply:
Seems like the issue is happening because of the missing `yq` command in the downstream.
I just tried locally and was able to reproduce this, let me update the d/s script.